src/Component.ts

```diff
@@ -48,6 +48,25 @@ export class Component<T extends HTMLElement = HTMLElement> extends BaseComponen
         return new Component<T>(document.createRange().createContextualFragment(html).children[0] as T);
     }
 
+    /**
+     * `Element#querySelector` shorthand
+     * @param selector A group of selectors
+     */
+    public select<T extends HTMLElement>(selector: string): Component<T> | null {
```

Comment:
make the type param optional
```suggestion
    public select<T extends HTMLElement = HTMLElement>(selector: string): Component<T> | null {
```